MapboxDirections/MBComponent.swift

```diff
@@ -1,45 +1,16 @@
 import Foundation
-
 /**
-The parent component that comprises what should be displayed in the instruction banner.
+The component protocol that comprises what the instruction banner should display.
  */
 @objc(MBComponent)
-open class Component: NSObject, NSSecureCoding {
-    
-    open static var supportsSecureCoding: Bool = true
-    
+public protocol Component: class, NSSecureCoding {
```

Comment:
Component is a pretty generic term, and a protocol’s name should describe some quality about its conforming classes. Since the two conforming classes are VisualInstructionComponent and LaneIndicationComponent, both of which are intended for visual display, let’s call the protocol something like Visualizable. (This isn’t a great name – any better ideas?)

Reply:
Probably, something like `ComponentRepresentable`?